bukuserver/templates/bukuserver/lib.html

```diff
@@ -23,6 +23,27 @@
   <script>opener && (opener !== window) && document.body.classList.add('popup')</script>
 {% endmacro %}
 
+{% macro brand_dbname(nohover=False) %}
+  <style>
+    .dbname {font-weight: bold;  font-family: monospace;  cursor: help}
+    .navbar-brand .dbname {font-size: smaller}
+    body:not(.popup) .navbar-brand {text-align: center}
+    body.popup .navbar-brand .dbname, body:not(.popup) .popup.dbname {display: none}
+    .popup.dbname {float: right;  padding: 15px 20px}
+    {% if nohover %}
+    body:not(.popup) .navbar-brand {padding: 5px 15px}
+    {% else %}
+    body:not(.popup) .navbar-brand:hover {padding: 5px 15px}
+    body:not(.popup) .navbar-brand:not(:hover) .dbname {display: none}
```

Comment:
"Hidden unless hovered" is not used in the bookmarklet popup because:
- bookmarklet popup makes it easier to accidentally add a bookmark into a wrong DB (so it's better to show DB name by default)
- mouse pointer events are disabled for the logo in the bookmarklet popup (to prevent accidentally opening the home page from a popup window)
- there's plenty of space in the navbar anyway (since all navigation elements get hidden in a popup window)